scripts/functions.py

```diff
@@ -137,170 +137,166 @@ def daylightSavings(inData):
 
     return inData
 
-def getPWRtechs(regions, techs):
+def getPWRtechs(region, techs):
     # PURPOSE: Creates all the PWR naming technologies
-    # INPUT:   regions =  Dictionary holding region as the key and subregion as the values in a list
+    # INPUT:   regions =  Tuple holding country as the key and subregion as the values in a dictionary
+    #                     (CAN, {WS:[...], MW:[]...},)

```

Comment:
Nitpick: You changed regions -> region in the function parameters but region -> regions in the function description! The same typo is in all of the similar functions.